content/source/docs/enterprise-beta/private/config.html.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Configuration"
+sidebar_current: "docs-enterprise2-private-config"
+---
+
+# Private Terraform Enterprise Configuration
+
+After you have completed the installation process you will need to create an
+administrator account, configure your Private Terraform Enterprise installation so
+it can send emails and SMS, and connect to your VCS provider.
+
+When the system configuration has been completed, create your first organizations and users and enable the
+enterprise features for those accounts.
+
+~> **Note:** If you are performing an upgrade or restore for an existing
+installation you _do not_ need to follow these steps. If your upgraded or
+restored installation does not function without the steps below then it was not
+correctly restored from backup. Please contact HashiCorp for help.
+
+This page covers the following topics:
+
+1. [System Configuration](#system-configuration)
+	1. [Create the administrator account](#creating-an-administrator)
+	2. [Configure SMTP for sending emails](#configure-smtp)
+	3. [Configure Twilio for sending SMS messages](#configure-twilio)
+	4. [Configure connectivity to git](#configure-git-hub-)
+	5. [Configure acceptance tests](#configure-acceptance-tests-verify-and-complete)
+2. [Team Setup](#team-setup)
+	1. [Inviting Colleagues](#inviting-colleagues)
+	2. [Creating an Organization](#creating-an-organization)
+	3. [Creating a Team](#creating-a-team)
+	4. [Enabling Enterprise Features](#enabling-enterprise-features)
+	5. [Accessing Terraform and Packer Features](#accessing-terraform-and-packer-features)
+	6. [Creating Additional Administrators](#creating-additional-administrators)
+
+## System Configuration
+
+In all examples below, be sure to replace "`<TFE HOSTNAME>`" with the hostname of your private Terraform Enterprise instance.
```

Comment:
I pushed some minor changes, and one of the big ones was around placeholders and UI text. We've been standardizing on `<ALLCAPS ANGLEBRACKETS>` for placeholder tokens, since they stand out visually and prevent a user from pasting or clicking text that isn't applicable to their install. And we've been using quotation marks to indicate UI-like text, while code spans indicate code-like or command-like text.